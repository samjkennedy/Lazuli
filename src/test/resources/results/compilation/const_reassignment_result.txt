Value `i: Int` is constant and cannot be reassigned:
2:
3:     i = 2
4:

Value `j: Int` is constant and cannot be reassigned:
5:     for (j: Int = 0 to 10) {
6:         j = 2
7:     }

