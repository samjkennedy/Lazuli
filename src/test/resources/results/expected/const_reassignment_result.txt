Value `i: Int` is constant and cannot be reassigned:
1:
2: i = 2
3:

Value `j: Int` is constant and cannot be reassigned:
4: for (j: Int = 0 to 10) {
5:     j = 2
6: }
